assets/test_10_examples_output.json

```diff
@@ -1 +1 @@
-{"status": "ok", "predictions": [[{"toxic": 0.00027056349790655077, "severe_toxic": 0.00013631785986945033, "obscene": 0.0001533413160359487, "threat": 9.785518341232091e-05, "insult": 0.00016848655650392175, "identity_hate": 0.0001446576789021492}], [{"toxic": 0.9871750473976135, "severe_toxic": 0.02508201077580452, "obscene": 0.9588805437088013, "threat": 0.0023930463939905167, "insult": 0.05074318125844002, "identity_hate": 0.001398613560013473}], [{"toxic": 0.00025861128233373165, "severe_toxic": 0.0001511252048658207, "obscene": 0.0001645928277866915, "threat": 0.00010689132614061236, "insult": 0.00016623981355223805, "identity_hate": 0.00015565518697258085}], [{"toxic": 0.9948775172233582, "severe_toxic": 0.2725079357624054, "obscene": 0.992616593837738, "threat": 0.007914072833955288, "insult": 0.9375438690185547, "identity_hate": 0.6642374396324158}], [{"toxic": 0.07105009257793427, "severe_toxic": 0.000670658191666007, "obscene": 0.012765652500092983, "threat": 9.756391955306754e-05, "insult": 0.07760647684335709, "identity_hate": 0.008826197125017643}], [{"toxic": 0.00030613478156737983, "severe_toxic": 0.00011995469685643911, "obscene": 0.00016595520719420165, "threat": 8.416409400524572e-05, "insult": 0.00015255478501785547, "identity_hate": 0.000128884770674631}], [{"toxic": 0.00026876915944740176, "severe_toxic": 0.00013043853687122464, "obscene": 0.00015963830810505897, "threat": 9.139275789493695e-05, "insult": 0.00016217704978771508, "identity_hate": 0.00013325244071893394}], [{"toxic": 0.9862720966339111, "severe_toxic": 0.0022539084311574697, "obscene": 0.39340513944625854, "threat": 0.006481354124844074, "insult": 0.054453782737255096, "identity_hate": 0.0050447797402739525}], [{"toxic": 0.0002996214898303151, "severe_toxic": 0.00012248451821506023, "obscene": 0.0001698401611065492, "threat": 8.313858415931463e-05, "insult": 0.00015916636039037257, "identity_hate": 0.00012199857155792415}], [{"toxic": 0.00027270056307315826, "severe_toxic": 0.00012360524851828814, "obscene": 0.00015428184997290373, "threat": 8.767773397266865e-05, "insult": 0.00016053511353675276, "identity_hate": 0.0001414872967870906}]]}
+{"status": "ok", "predictions": [{"toxic": 0.9796434044837952, "severe_toxic": 0.07256636023521423, "obscene": 0.058431386947631836, "threat": 0.8635178804397583, "insult": 0.11121545732021332, "identity_hate": 0.013826466165482998}, {"toxic": 0.00029103411361575127, "severe_toxic": 0.00012417171092238277, "obscene": 0.0001522742968518287, "threat": 8.440738747594878e-05, "insult": 0.00016013195272535086, "identity_hate": 0.00012860879360232502}]}
```

Comment:
Just wondering why this is now only a single record? Previously it was 10?

Reply:
I think I made a mistake in the `curl` command so that it parsed the filename instead of the actual file.. I have updated this. Good find, thanks!